src/services/cartsService.js

```diff
@@ -19,4 +20,8 @@ const addItemToCartsService = async (userId, productId, sizeId) => {
   return false;
 };
 
-module.exports = { addItemToCartsService };
+const getCartsService = async (userId) => {
+  return await getProduct(userId);
```

Comment:
```suggestion
  return await getCarts(userId);
```
cart를 불러오는 것이니 네이밍 변경해주세요. 만약 정말 products라는 키워드가 꼭 필요하다고 생각이 든다면
`getProductsOfCarts`라고 할 수 있겠습니다